tests/testthat/indention_token_dependent/token_dependent_comments-out.R

```diff
@@ -0,0 +1,7 @@
+call(call( # comment
+  3, 4
+))
+
+call(call(1, # comment
+          3
```

Comment:
I'd rather have the `3` indented by two space by default, and only optionally support indention after the opening paren. I remember a discussion about this but can't currently find it.